Simplified/Book/Models/NYPLBookRegistry.h

```diff
@@ -142,6 +142,10 @@ genericBookmarks:(nullable NSArray<NYPLBookLocation *> *)genericBookmarks;
 // TODO: Remove when migrate to Swift, use setState:forIdentifier: instead
 - (void)setStateWithCode:(NSInteger)stateCode forIdentifier:(nonnull NSString *)identifier;
 
+// Reset the book state for the given book identifier to NYPLBookStateDownloadNeeded
+// for books that were in a downloaded/downloading states.
+- (void)resetStateToDownloadNeededForIdentifier:(nonnull NSString *)identifier;
```

Comment:
so this is odd.... I already see this change _currently_ on master:

https://github.com/NYPL-Simplified/Simplified-iOS/blob/cb2f4e4c9c6d5af61271d40351ff8ad7c2eb6991/Simplified/Book/Models/NYPLBookRegistry.h#L145-L148

Instead if I do the merge locally the merge commit does not show these changes... which honestly it seems more correct to me. Have you seen this before @ErnestFan ?